### ISSUE TITLE
got stucked when import torch

### ISSUE THREAD

**Issue by hongdayu** [triton-inference-server/server#3152]
**Description**
python backend, import torch in model.py, after initialize it will got stucked
![图片](https://user-images.githubusercontent.com/32477868/126989986-8cca8eef-0c30-40dd-bdc4-f0da20cb263e.png)

![图片](https://user-images.githubusercontent.com/32477868/126989855-6d9506b6-3859-4d3e-9cd8-fa0dd9baa2e9.png)

**Triton Information**
dockfile
```
FROM nvcr.io/nvidia/tritonserver:21.06.1-py3
COPY . .
RUN pip3 install --no-cache-dir -r requirements_server.txt & \
    pip3 install --no-cache-dir torch==1.8.1+cu111 torchvision==0.9.1+cu111 -f https://download.pytorch.org/whl/torch_stable.html
```

**Comment by Tabrizian** [comment]
We had a bug in the 21.06 release that Python models didn't work properly when there was an import that could take a long time. We have fixed this issue and the fix is going to be available in the 21.07 release. Can you try building from source and use "r21.07" as the repository tag and let us know whether it fixes your issue or not?
https://github.com/triton-inference-server/server/blob/main/docs/build.md#building-with-docker

**Comment by hongdayu** [comment]
> We had a bug in the 21.06 release that Python models didn't work properly when there was an import that could take a long time. We have fixed this issue and the fix is going to be available in the 21.07 release. Can you try building from source and use "r21.07" as the repository tag and let us know whether it fixes your issue or not?
> https://github.com/triton-inference-server/server/blob/main/docs/build.md#building-with-docker

Thank you for your answer . When was 21.07  released? I have no experience for building from source, I will try it.

**Comment by Tabrizian** [comment]
It is released today. Closing this issue, feel free to reopen if the issue persisted.
